src/pages/components/inline-loading/accessibility.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Inline Loading
+tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+---
+
+import {
+  StructuredListWrapper,
+  StructuredListHead,
+  StructuredListBody,
+  StructuredListRow,
+  StructuredListInput,
+  StructuredListCell,
+  OrderedList,
+  ListItem,
+} from 'carbon-components-react';
+
+<AnchorLinks>
+  <AnchorLink>How it works</AnchorLink>
+  <AnchorLink>Accessibility considerations</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Accessibility testing</AnchorLink>
+</AnchorLinks>
+
+## How it works
+
+The Carbon Inline Loading Component is used to notify user’s that their action is being processed. W3C WAI-ARIA role="alert" and aria-live="assertive" are used to ensure screen reader users are also provided the same information.
```

Comment:
```suggestion
The inline loading Carbon component is used to notify users that their action is being processed. W3C WAI-ARIA role="alert" and aria-live="assertive" are used to ensure screen reader users are also provided the same information.
```

Should `role="alert"` and `aria-live="assertive"` be code snippets?

Reply:
Made `aria-live="assertive"` and `role="alert"` code snippets and fixed the capitalization to be consistent with the rest of the site 👍 !